server/sheet.go

```diff
@@ -162,6 +162,8 @@ func (s *Server) registerSheetRoutes(g *echo.Group) {
 
 		sheetFind := &api.SheetFind{
 			ID: &id,
+			// Get the full statement by default when fetching sheet with id.
```

Comment:
We should not load full from frontend.

Reply:
Maybe we need to show full statement for the normal sheet from SQL Editor?